core/controlplane/config/config.go

```diff
@@ -1317,13 +1304,11 @@ func (c Cluster) validate() error {
 		}
 	}
 
-	if c.Kubernetes.Networking.SelfHosting.Enabled {
-		if (c.Kubernetes.Networking.SelfHosting.Type != "canal") && (c.Kubernetes.Networking.SelfHosting.Type != "flannel") {
-			return fmt.Errorf("networkingdaemonsets - style must be either 'canal' or 'flannel'")
-		}
-		if c.Kubernetes.Networking.SelfHosting.Typha && c.Kubernetes.Networking.SelfHosting.Type != "canal" {
-			return fmt.Errorf("networkingdaemonsets - you can only enable typha when deploying type 'canal'")
-		}
+	if (c.Kubernetes.Networking.SelfHosting.Type != "canal") && (c.Kubernetes.Networking.SelfHosting.Type != "flannel") {
```

Comment:
Bit of a picky comment, but can we keep if statement style consistent.

`if (c.Kubernetes.Networking.SelfHosting.Type != "canal") && (c.Kubernetes.Networking.SelfHosting.Type != "flannel"){` can be replaced by
` if c.Kubernetes.Networking.SelfHosting.Type != "canal" && c.Kubernetes.Networking.SelfHosting.Type != "flannel"` right?

Reply:
Sure - that makes sense, have updated!